playbooks/common/openshift-cluster/redeploy-certificates/ca.yml

```diff
@@ -294,7 +294,7 @@
       client_path: "{{ openshift.common.config_base }}/master/admin.kubeconfig"
       ca_path: "{{ openshift.common.config_base }}/master/ca-bundle.crt"
   - name: Lookup default group for ansible_ssh_user
-    command: "/usr/bin/id -g {{ ansible_ssh_user }}"
+    command: "/usr/bin/id -g {{ ansible_ssh_user | quote }}"
```

Comment:
@sdodson I think it might not, it looks like `pwd` is tied to the passwd database...  I'm wondering if there is a better option, though